src/models/UserToken.ts

```diff
@@ -0,0 +1,27 @@
+interface UserToken {
+  groups: string[];
+  name: string;
+  accessToken: string;
+  expires: Date;
+  refreshToken: string;
+  refreshExpires: Date;
+}
+
+const userTokenBuilder = (userToken: UserToken): UserToken => {
```

Comment:
Tar inn en "UserToken" for så å mappe til "UserToken"?

Reply:
For at expiry-feltene skal mappes til faktiske datoer, men skulle vel egentlig stått at man mappet fra `any` eller noe sånt.

Reply:
Laga en egen serialisert klasse for å skille så ble det mer tydelig hva som skjer.